cmake/modules/hunter_download.cmake

```diff
@@ -429,7 +429,7 @@ function(hunter_download)
   file(
       APPEND
       "${HUNTER_DOWNLOAD_TOOLCHAIN}"
-      "set(HUNTER_KEEP_PACKAGE_SOURCES \"${HUNTER_KEEP_PACKAGE_SOURCES}\" CACHE INTERNAL \"\")\n"
+      "set(HUNTER_KEEP_PACKAGE_SOURCES \"${_hunter_keep_package_sources}\" CACHE INTERNAL \"\")\n"
```

Comment:
Variable `_hunter_keep_package_source` is not defined at this point.

Reply:
All packages that depends on this package will be build with `HUNTER_KEEP_PACKAGE_SOURCES=ON` even if you want to keep sources only for one package. TL;DR I think this should be reverted.

Reply:
Reverted (will submit clean pr once it is working)